### ISSUE TITLE
Weird looking outputs with random artificats/patterns

### ISSUE THREAD

**Issue by its-nmt05** [dome272/Diffusion-Models-pytorch#46]
After training the model for around 1000 epochs, I am getting weird patterns/artifacts as output instead of meaningful images. I have not been able to trace out any particular reason for this issue yet. PS. Will update the issue if I find one.

Take a look at some of the outputs here 👇 
## With GroupNorm
![image_e50_s1](https://github.com/user-attachments/assets/bd42efd8-530c-439d-aff2-31be0a41830d) ![image_e50_s0](https://github.com/user-attachments/assets/01ad9095-7c12-450f-b731-804af092ff4b)
## with batchNorm
![image_e180_s1](https://github.com/user-attachments/assets/28809cd9-0707-48e3-8376-67ded68f3a4f) ![image_e110_s1](https://github.com/user-attachments/assets/8a32077d-47e1-4f30-90b2-de032fc18af6) ![image_e50_s0](https://github.com/user-attachments/assets/2dd82f0d-7a7b-4395-b871-d846c4addc7c)

You can find all the code here: https://github.com/its-nmt05/DeepLLMs/tree/main/diffusion
